cpp/tests/hashing/hash_test.cpp

```diff
@@ -280,31 +295,38 @@ TEST_F(SparkMurmurHash3Test, MultiValueWithSeeds)
 {
   // The hash values were determined by running the following Scala code in Apache Spark:
   // import org.apache.spark.sql.catalyst.util.DateTimeUtils
-  // val schema = new StructType().add("strings",StringType).add("doubles",DoubleType)
-  //   .add("timestamps",TimestampType).add("decimal64", DecimalType(18,7)).add("longs",LongType)
-  //   .add("floats",FloatType).add("dates",DateType).add("decimal32", DecimalType(9,3))
-  //   .add("ints",IntegerType).add("shorts",ShortType).add("bytes",ByteType)
-  //   .add("bools",BooleanType)
+  // val schema = new StructType().add("structs", new StructType().add("a",IntegerType)
+  //     .add("b",StringType).add("c",new StructType().add("x",FloatType).add("y",LongType)))
+  //   .add("strings",StringType).add("doubles",DoubleType).add("timestamps",TimestampType)
+  //   .add("decimal64", DecimalType(18,7)).add("longs",LongType).add("floats",FloatType)
+  //   .add("dates",DateType).add("decimal32", DecimalType(9,3)).add("ints",IntegerType)
+  //   .add("shorts",ShortType).add("bytes",ByteType).add("bools",BooleanType)
```

Comment:
Can we delete this stale code or is it here for a good reason?

Reply:
This isn't stale code, it's part of the comment showing the Scala script used to produce/verify the hashed test values.  If someone comes along later and wonders how to update the hash values for a new type or new test values for an existing type, this helps point them in the right direction.